proguard/src/main/java/com/flurry/proguard/UploadMapping.java

```diff
@@ -197,13 +198,15 @@ private static File createArchive(List<File> files, String uuid) {
      * @param token the Flurry auth token
      * @return the project's ID
      */
-    private static String lookUpProjectId(String apiKey, String token) {
+    private static String lookUpProjectId(String apiKey, String token) throws IOException {
         String queryUrl = String.format("%s/project?fields[project]=apiKey&filter[project.apiKey]=%s",
                     METADATA_BASE, apiKey);
-        HttpResponse response = executeHttpRequest(new HttpGet(queryUrl), getMetadataHeaders(token));
-        expectStatus(response, HttpURLConnection.HTTP_OK);
+        JSONObject jsonObject;
+        try (CloseableHttpResponse response = executeHttpRequest(new HttpGet(queryUrl), getMetadataHeaders(token))) {
+            expectStatus(response, HttpURLConnection.HTTP_OK);
+            jsonObject = getJsonFromEntity(response.getEntity());
+        }
 
-        JSONObject jsonObject = getJsonFromEntity(response.getEntity());
         JSONArray jsonArray = jsonObject.getJSONArray("data");
```

Comment:
Since the content of the jsonArray is dependent on the response , I guess we can move these lines to be inside the try block